spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/DefaultPollerProperties.java

```diff
@@ -60,4 +80,19 @@ public void setMaxMessagesPerPoll(long maxMessagesPerPoll) {
 		this.maxMessagesPerPoll = maxMessagesPerPoll;
 	}
 
+	public String getCron() {
+		return cron;
```

Comment:
Don't we need to have a `this.` prefix whenever we access to class properties?
I'm not sure what is your Checkstyle config, but it is not allowed without prefix in my projects.

Thanks